server/server.go

```diff
@@ -486,19 +451,12 @@ func (j *jsonRPCHub) GetForksInTime(blockNumber uint64) chain.ForksInTime {
 }
 
 func (j *jsonRPCHub) GetStorage(root types.Hash, addr types.Address, slot types.Hash) ([]byte, error) {
-	account, err := j.GetAccount(root, addr)
-
-	if err != nil {
-		return nil, err
-	}
-
-	obj, err := j.getState(account.Root, slot.Bytes())
-
+	snap, err := j.state.NewSnapshotAt(root)
 	if err != nil {
 		return nil, err
```

Comment:
shall we add some context to the error? maybe root hash and address?

Reply:
`NewSnapshotAt` already returns in the error message the root that cannot be found.

Reply:
do we have context what root has failed?

Reply:
Fixed in 8c5f05599efa28b2a31c9a93db2878d7cc7ddd6a